server/ai_mediaserver.go

```diff
@@ -452,6 +455,48 @@ func (ls *LivepeerServer) StartLiveVideo() http.Handler {
 	})
 }
 
+func (ls *LivepeerServer) UpdateLiveVideo() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		ctx := r.Context()
+		if r.Method != http.MethodPost {
+			http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
+			return
+		}
+		// Get stream from path param
+		stream := r.PathValue("stream")
+		if stream == "" {
+			http.Error(w, "Missing stream name", http.StatusBadRequest)
+			return
+		}
+		ls.LivepeerNode.LiveMu.RLock()
+		defer ls.LivepeerNode.LiveMu.RUnlock()
+		p, ok := ls.LivepeerNode.LivePipelines[stream]
+		if !ok {
+			// Stream not found
+			http.Error(w, "Stream not found", http.StatusNotFound)
+			return
+		}
+		defer r.Body.Close()
+		params, err := io.ReadAll(r.Body)
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+
+		clog.V(6).Infof(ctx, "Sending Live Video Update Control API stream=%s, params=%s", stream, string(params))
+		if err := p.ControlPub.Write(strings.NewReader(string(params))); err != nil {
```

Comment:
Are we also planning on returning a response from the inference process?

If so, can be a separate PR. As the doc mentions, that would be a more involved process with a separate control response subscription. The params would also need to be wrapped in some sort of transaction id to reconcile request/response.

However, if we _don't_ need a response then we can keep things as-is since that is much simpler.

Reply:
Yeah, good point. I think we can leave it without a response for now. 

But in the future, we'll probably need a way to signal if the update failed for some reason.